### PR TITLE
[TASK] Clean up code in CrawlerController

### PR DESCRIPTION
**Pull request by bmack**
- Use proper (int) type-casting instead of intval() = faster
- Use $this->tableName instead of 'tx_crawler_queue'
- Use $this->backendUser instead of $GLOBALS[BE_USER]
- Remove unused HTML styling code
- Move CLI_checkIfProcessIsActive into ProcessRepository

### CODE REVIEW

File: Classes/Domain/Repository/ProcessRepository.php [hunk 1]
```diff
@@ -296,4 +296,20 @@ public function deleteProcessesMarkedAsDeleted()
             )
             ->execute();
     }
+
+    public function isProcessActive(int $processId): bool
+    {
+        $queryBuilder = GeneralUtility::makeInstance(ConnectionPool::class)->getQueryBuilderForTable($this->tableName);
+        $isActive = $queryBuilder
+            ->select('active')
+            ->from('tx_crawler_process')
```
Comment (tomasnorre) [inline Classes/Domain/Repository/ProcessRepository.php]: `$this->tableName` ?  it's already used two lines above.

File: Classes/Domain/Repository/ProcessRepository.php [hunk 2]
```diff
@@ -296,4 +296,20 @@ public function deleteProcessesMarkedAsDeleted()
...
+            ->where(
+                $queryBuilder->expr()->eq('process_id', $queryBuilder->createNamedParameter($processId, \PDO::PARAM_INT))
```
Comment (tomasnorre) [inline Classes/Domain/Repository/ProcessRepository.php]: The `process_id` is not an integer.. 
https://github.com/AOEpeople/crawler/pull/462/files#diff-481209dc0ee6be420867b4ac30992471R1980

File: Classes/Controller/CrawlerController.php [hunk 3]
```diff
@@ -1360,12 +1361,12 @@ public function readUrlFromArray($field_array)
     {
             // Set exec_time to lock record:
         $field_array['exec_time'] = $this->getCurrentTime();
-        $connectionForCrawlerQueue = GeneralUtility::makeInstance(ConnectionPool::class)->getConnectionForTable('tx_crawler_queue');
+        $connectionForCrawlerQueue = GeneralUtility::makeInstance(ConnectionPool::class)->getConnectionForTable($this->tableName);
```
Comment (tomasnorre) [inline Classes/Controller/CrawlerController.php]: I know I started this code, but perhaps it should be `$this->tableCrawleQueue` instead, as we might end up with more tableNames later. 

But for now it isn't a problem and can be refactored later.

File: Classes/Domain/Repository/ProcessRepository.php (hunk 1 above)
Comment (bmack) [reply Classes/Domain/Repository/ProcessRepository.php]: correct. will fix

File: Classes/Domain/Repository/ProcessRepository.php (hunk 2 above)
Comment (tomasnorre) [reply Classes/Domain/Repository/ProcessRepository.php]: Ups.. Then it's on me. But shouldn't work.

Comment (bmack) [reply Classes/Domain/Repository/ProcessRepository.php]: fixed.

**Pull request merged**